packages/server/email/teamInviteEmailCreator.tsx

```diff
@@ -3,7 +3,7 @@ import TeamInvite, {TeamInviteProps} from 'parabol-client/modules/email/componen
 import {headCSS} from 'parabol-client/modules/email/styles'
 import React from 'react'
 
-const subject = 'You’ve been invited to Parabol'
+const subject = (inviterEmail: string) : string => `${inviterEmail} has invited you to Parabol`
```

Comment:
-1 I think the inviter name would be nicer to read here

Reply:
Resolved the request!